### PR TITLE
add LibPspExploit

### PR DESCRIPTION
**Pull request by bucanero**

### CODE REVIEW

File: libpspexploit/PSPBUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
+pkgname=libpspexploit
+pkgver=1.2
+pkgrel=1
+pkgdesc="LibPspExploit is a library that allows creating OFW-compatible homebrew with kernel access."
+arch=('mips')
+url="https://github.com/PSP-Archive/LibPspExploit/"
+license=('Unknown')
```
Comment (bucanero) [inline libpspexploit/PSPBUILD:7]:
```suggestion
license=('WTFPL')
```

**Review by diamant3 — CHANGES_REQUESTED** [review]
thanks! @bucanero but could you add the license to package too

**Comment by bucanero** [comment]
> thanks! @bucanero but could you add the license to package too

license added 👍

File: libpspexploit/PSPBUILD [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "https://github.com/PSP-Archive/LibPspExploit/archive/refs/tags/v${pkgver}.zip"
+    "http://sam.zoy.org/wtfpl/COPYING"
+    )
+sha256sums=(
+    'SKIP'
+    'SKIP'
+    )
+
+prepare() {
+    cp COPYING "${srcdir}/LibPspExploit-${pkgver}/LICENSE"
```
Comment (diamant3) [inline libpspexploit/PSPBUILD:21]: and then this one, you could rename the LICENSE.txt from the source to LICENSE

**Review by diamant3 — COMMENTED** [review]
After that, I will merge it

**Comment by diamant3** [comment]
Thanks!

**Pull request merged**